### PR TITLE
feat(blackbox): change image location

### PR DESCRIPTION
**Pull request by georgettica**
now will pull from quay and not dockerhub

### CODE REVIEW

**Comment by georgettica** [comment]
something interesting
https://quay.io/repository/app-sre/prom-blackbox-exporter?tab=tags
and
https://hub.docker.com/r/prom/blackbox-exporter/tags?page=1&ordering=last_updated

for inconsistencies

**Comment by maorfr** [comment]
I would suggest to template the image out so it can be configurable.

**Comment by georgettica** [comment]
thanks! I will see how I will implement that :)

**Comment by georgettica** [comment]
made to WIP as the change now needs to be tested

**Comment by georgettica** [comment]
WDYT @maorfr ?

**Comment by maorfr** [comment]
I'm not sure why the code needs to know the image we plan to use... I don't want to throw things in the wrong direction and would rather wait for your team to review.

**Comment by georgettica** [comment]
anyways, it works :)
if it's not required it will be removed by the team tomorrow

**Comment by georgettica** [comment]
/hold
it works but waiting for review

**Review by RiRa12621 — APPROVED** [review]
/lgtm
/approve

**Comment by georgettica** [comment]
Done in browser, not sure if looks ok

File: controllers/routemonitor/routemonitor.go [hunk 1]
```diff
@@ -41,13 +40,12 @@ type RouteMonitorReconciler struct {
 	RouteMonitorDeleter
 }
 
-// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;delete
 // +kubebuilder:rbac:groups=*,resources=services,verbs=get;list;watch;create;delete
+// +kubebuilder:rbac:groups=apps,resources=deployments,verbs=get;list;watch;create;delete
```
Comment (georgettica) [inline controllers/routemonitor/routemonitor.go:44]: Might be sorting related, will check tomorrow

**Pull request merged**